plugins/block-dynamic-connection/src/dynamic_text_join.js

```diff
@@ -69,6 +82,21 @@ Blockly.Blocks['dynamic_text_join'] = {
     this.inputCounter = next;
   },
 
+  /**
+   * Parses XML based on the 'items' attribute (standard).
+   * @param {!Element} xmlElement XML storage element.
+   * @this {Blockly.Block}
+   */
+  deserializeCounts_: function(xmlElement) {
+    const itemCount = Math.max(
+        parseInt(xmlElement.getAttribute('items'), 10), this.minInputs);
+    // Two inputs are added automatically.
+    for (let i = 2; i < itemCount; i++) {
```

Comment:
Can we use `this.minInputs` here?

Reply:
We can, but it's a bit weird because `minInputs` not being used [by what actually determines what this number needs to be](https://github.com/google/blockly-samples/blob/3832abc63faaf5fe5d40a2e4c2dcfea90267e391/plugins/block-dynamic-connection/src/dynamic_text_join.js#L34). So it kinda felt right to keep it consistently magical? But saying it out loud seems weird lol

I'll switch it.

Reply:
Ah, I see what you mean. 

Yeah that should probably be cleaned up a bit, but we can do that in a separate PR. This fix LGTM.

Thanks!